__tests__/schema/uuid/abstract-navigation-child.ts

```diff
@@ -281,200 +321,228 @@ describe('Page', () => {
             },
           },
         },
-      },
-      client,
-    })
+      })
   })
-})
 
-describe('Taxonomy Term', () => {
-  test('Without navigation', async () => {
-    const client = createTestClient({
-      service: Service.Serlo,
-      userId: null,
-    })
-    await assertSuccessfulGraphQLMutation({
-      ...createSetTaxonomyTermMutation(taxonomyTermRoot),
-      client,
-    })
-    await assertSuccessfulGraphQLMutation({
-      ...createSetTaxonomyTermMutation(taxonomyTermSubject),
-      client,
-    })
-    await assertSuccessfulGraphQLMutation({
-      ...createSetNavigationMutation({
-        instance: Instance.De,
-        data: [],
-      }),
-      client,
-    })
-    await assertSuccessfulGraphQLQuery({
-      query: gql`
-        query uuid($id: Int!) {
-          uuid(id: $id) {
-            ... on TaxonomyTerm {
-              navigation {
-                path {
-                  nodes {
-                    label
-                    url
-                    id
+  describe('Taxonomy Term', () => {
+    // Der Test hier wirft den Fehler:
+    // console.error
+    // Found an unhandled request for message {"type":"UuidQuery","payload":{"id":5}}
+
+    test('Without navigation', async () => {
+      const client = new Client({ service: Service.Serlo })
+
+      await client
+        .prepareQuery(createSetTaxonomyTermMutation(taxonomyTermRoot).mutation)
+        .withVariables(
+          createSetTaxonomyTermMutation(taxonomyTermRoot).variables
+        )
+        .execute()
+
```

Comment:
```suggestion
      await client
        .prepareQuery(createSetTaxonomyTermMutation(taxonomyTermSubject).mutation)
        .withVariables(
          createSetTaxonomyTermMutation(taxonomyTermSubject).variables
        )
        .execute()
```
The test fails because you are missing this code.